CRM/Core/SelectValues.php

```diff
@@ -568,14 +568,7 @@ public static function contributionTokens(): array {
     foreach ($processor->getAllTokens() as $token => $title) {
       $tokens['{contribution.' . $token . '}'] = $title;
     }
-    return array_merge($tokens, [
-      '{contribution.cancel_reason}' => ts('Contribution Cancel Reason'),
-      '{contribution.amount_level}' => ts('Amount Level'),
-      '{contribution.check_number}' => ts('Check Number'),
-      '{contribution.campaign}' => ts('Contribution Campaign'),
```

Comment:
Note I manually tested that this token still works in the original legacy flow - although it is no longer advertised and it never worked in the scheduled reminders flow. This change just stops it from being advertised